src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +99,29 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('save_api_credentials', False):
+            # json object for variables file
+            # set hostname to stdin or env variable
+            if options.get('hostname', False):
+                hostname = options.get('hostname')
+            else:
+                hostname = os.environ['PYTEST_BASE_URL']
```

Comment:
This won't be a hostname, it will be a base URL. If we want the hostname we should parse the URL.